### PR TITLE
Prototype Banner Slider

### PR DESCRIPTION
**Pull request by dfosco**
Style adjustments for current iteration of banner slider

### CODE REVIEW

**Comment by dfosco** [comment]
@tamorim @amoreira pls revew

File: src/components/Banner/index.js [hunk 1]
```diff
@@ -17,7 +17,22 @@ class Banner extends React.Component {
...
+      },
+      {
+        url: 'http://i.imgur.com/qk1PCBR.jpg',
+        link: 'http://www.google.com',
+        alt: 'Default image'
+      }
+
+    ];
+    let images = defaultImage;
+    //mock-end
```
Comment (tamorim) [inline src/components/Banner/index.js]: @dfosco Please remove the mock before merging

**Comment by dfosco** [comment]
@tamorim Done

**Comment by dfosco** [comment]
@tamorim Not done! Hold up, I broke some stuff, haha

**Comment by dfosco** [comment]
@tamorim All rigtht, good to go :+1:

**Pull request closed without merging**